charts/merlin/values.yaml

```diff
@@ -38,6 +38,11 @@ mlpApi:
 feastCoreApi:
   apiHost: http://feast-core.mlp:8080/v1
 imageBuilder:
+  serviceAccount:
+    create: true
+    name: "kaniko-sa"
```

Comment:
Should we set this by default? Setting name to "" will revert to old logic of mounting kaniko secret into pod (since clusters may not have WI enabled)

Reply:
Why not use the `create` flag instead of `name` in the deployment? to add the SA to deploy here: https://github.com/caraml-dev/helm-charts/pull/214/files#diff-aa40967bd19f1d928b7210f0d5deff4d93f9e01eb523c0a6c15e37fbe40b54d4R162

Reply:
@maniteja6799 we may have kaniko SA provided. So we don't need to create it but still need to pass the name in env variable